hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/freon/DNRPCLoadGenerator.java

```diff
@@ -91,6 +97,16 @@ public class DNRPCLoadGenerator extends BaseFreonGenerator
       defaultValue = "1")
   private int numClients = 1;
 
+  @Option(names = {"--read-only"},
+      description = "if Ratis, read only or not",
+      defaultValue = "false")
+  private boolean readOnly = false;
+
+  @Option(names = {"--ratis"},
```

Comment:
Can we treat absence of `--ratis` as `--read-only`?

Reply:
Pushed a commit to force read only if in grpc mode.